sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -83,23 +83,24 @@ object Blinding {
 
   private[engine] def partyPackages(
       tx: VersionedTransaction,
-      blindingInfo: BlindingInfo,
+      disclosure: Relation[NodeId, Party],
+      contractVisibility: Relation[ContractId, Party],
       contractPackages: Map[ContractId, Ref.PackageId],
   ): Relation[Party, Ref.PackageId] = {
     Relation.from(
-      disclosedPartyPackages(tx, blindingInfo.disclosure) ++
-        divulgedPartyPackages(contractPackages, blindingInfo.divulgence)
+      disclosedPartyPackages(tx, disclosure) ++
```

Comment:
Looking at `disclosedPartyPackages`, it seems to disregard interface ids in fetch nodes. Could you please double-check if they need to be added?

Reply:
> Since we disallow retroactive instance, I think it is fine to ignore interface Ids in practice.

Please let's not rely on that.

> However I think it is safer to include them, in case we have to change something.

Yes, please.